src/utils/helpers.utils.js

```diff
@@ -194,3 +194,14 @@ export const isArticleExist = async articleId => {
   if (getArticle) return getArticle;
   return 'article does not exist';
 };
+
+/**
+ * Check Article existence
+ *
+ * @param {String} slug
+ * @returns {Boolean} true if email exists
+ * @returns {Boolean} false if email does not exist
+ */
+export const findArticleBySlug = async slug => {
```

Comment:
Am not sure we are using this anywhere in this PR, if not we should remove it.

Reply:
The initial plan was to use the slug to as the param but that feature had not been merged yet so I had to refactor to use the getArticleById... It would be changed again to handle the slug afterwards. But would you you still suggest that it's taken out?